examples/integration/iOSApp/Source/CoreUtilsMixed/MixedAnswer/BUILD

```diff
@@ -0,0 +1,58 @@
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library",
+)
+load(
+    ":module_map.bzl",
+    "module_map",
+)
+
+module_map(
+    name = "MixedAnswer_objc_modulemap",
+    hdrs = ["MixedAnswer.h"],
+    module_name = "MixedAnswer",
+)
+
+module_map(
+    name = "MixedAnswer_swift_modulemap",
+    hdrs = ["MixedAnswer.h"],
+    module_name = "MixedAnswer",
+    deps = [
+        ":MixedAnswerLib_Swift",
+    ],
+)
+
+objc_library(
+    name = "MixedAnswer",
+    srcs = ["MixedAnswer.m"],
+    hdrs = ["MixedAnswer.h"],
+    copts = [
+        "-I$(BINDIR)/iOSApp/CoreUtilsMixed",
+    ],
+    module_name = "MixedAnswer",
+    visibility = ["//iOSApp:__subpackages__"],
+    deps = [
+        ":MixedAnswerLib_Swift",
+        ":MixedAnswer_objc_modulemap",
+        ":MixedAnswer_swift_modulemap",
+    ],
+)
+
+swift_library(
+    name = "MixedAnswerLib_Swift",
+    srcs = ["MixedAnswer.swift"],
+    copts = [
+        "-Xfrontend",
+        "-import-underlying-module",
```

Comment:
We currently don't support `-Xfrontend` flags. It's the main blocker for supporting rules_ios at this moment. I'm working to address that in this release.

Please leave this PR open and I'll close it when I have this working in BwX mode.

Reply:
Wonderful, will do. Thanks again